Doc/library/profile.rst

```diff
@@ -148,14 +148,15 @@ entries according to the standard module/line/name string that is printed. The
 :meth:`~pstats.Stats.print_stats` method printed out all the statistics.  You
 might try the following sort calls::
 
-   p.sort_stats('name')
+   p.sort_stats(SortKey.name)
    p.print_stats()
 
 The first call will actually sort the list by function name, and the second call
 will print out the statistics.  The following are some interesting calls to
 experiment with::
 
    p.sort_stats('cumulative').print_stats(10)
+   p.sort_stats(SortKey.cumulative).print_stats(10)
```

Comment:
I still think showing both ways in these examples is not useful and could create confusion.
It would be enough to have one example here and the changes that you already did in the method docs later in this file.

Reply:
Ok. unless there're objections from anyone else,  I'll remove them and just keep the one in the method doc.